src/event.ts

```diff
@@ -1,12 +1,13 @@
 export interface Event {
     payload?: string;
-    action?: string;
     connKey?: number[];
     dstPort: number;
     rule?: string;
+    handler?: string;
     scanner?: string;
     sensorID: string;
     srcHost: string;
     srcPort: string;
     timestamp: string;
+    decoded?: any;
```

Comment:
Could you explain what these two fields represent? Should we add a description comment for each field in this file?

Reply:
Yes, that makes sense :+1: